lib/puppet/pops/loader/ruby_data_type_instantiator.rb

```diff
@@ -0,0 +1,40 @@
+# The RubyTypeInstantiator instantiates a data type from the ruby source
+# that calls Puppet::DataTypes.create_type.
+#
+class Puppet::Pops::Loader::RubyDataTypeInstantiator
+  # Produces an instance of the AnyType class with the given typed_name, or fails with an error if the
+  # given ruby source does not produce this instance when evaluated.
+  #
+  # @param loader [Puppet::Pops::Loader::Loader] The loader the type is associated with
+  # @param typed_name [Puppet::Pops::Loader::TypedName] the type / name of the type to load
+  # @param source_ref [URI, String] a reference to the source / origin of the ruby code to evaluate
+  # @param ruby_code_string [String] ruby code in a string
+  #
+  # @return [Puppet::Pops::Types::AnyType] - an instantiated data type associated with the given loader
```

Comment:
"an instantiated data type (subtype of AnyType), ..." ?

Reply:
The type of the return value is stated within brackets. Why duplicate that in the description?